avalon/nuke/workio.py

```diff
@@ -42,6 +42,11 @@ def current_file():
     return os.path.normpath(current_file).replace("\\", "/")
 
 
-def work_root():
-    from avalon import Session
-    return os.path.normpath(Session["AVALON_WORKDIR"]).replace("\\", "/")
+def work_root(session):
+
+    if scene_dir:
+        path = os.path.join(work_dir, scene_dir)
```

Comment:
I believe that `scene_dir` and `work_dir` are undefined. :wink:

Reply:
Thanks, well spotted. Where is the Hound? 🐶

Reply:
Fixed with: https://github.com/getavalon/core/pull/442/commits/83617b63544ccb0336a8afc2726fd2e8dfacb69f